app/src/main/java/com/kamron/pogoiv/OcrHelper.java

```diff
@@ -509,12 +509,12 @@ private String getPokemonIdentifierFromImg(Bitmap pokemonImage) {
             return Optional.absent();
         }
 
-        Bitmap candyAmount = getImageCrop(pokemonImage, 0.65, 0.7, 0.15, 0.035);
+        Bitmap candyAmount = getImageCrop(pokemonImage, 0.60, 0.7, 0.20, 0.035);
```

Comment:
i'm proposing we go up a bit and higher and more height
getImageCrop(pokemonImage, 0.60, 0.695, 0.20, 0.038)
![modified image1](https://cloud.githubusercontent.com/assets/22281828/20452791/0522119c-ade0-11e6-90b9-d33366790008.png)

PR value
![changed](https://cloud.githubusercontent.com/assets/22281828/20452794/15c2eef4-ade0-11e6-9b47-38e13749c28c.png)